stubs/netaddr/netaddr/ip/__init__.pyi

```diff
@@ -148,14 +148,14 @@ class IPRange(BaseIP, IPListMixin):
     def cidrs(self) -> list[IPNetwork]: ...
 
 def iter_unique_ips(*args: IPRange | _IPNetworkAddr) -> Iterator[IPAddress]: ...
-def cidr_abbrev_to_verbose(abbrev_cidr: str | SupportsInt | SupportsIndex) -> str: ...
+def cidr_abbrev_to_verbose(abbrev_cidr: ConvertibleToInt) -> str: ...
 def cidr_merge(ip_addrs: Iterable[IPRange | _IPNetworkAddr]) -> list[IPNetwork]: ...
 def cidr_exclude(target: _IPNetworkAddr, exclude: _IPNetworkAddr) -> list[IPNetwork]: ...
 def cidr_partition(
     target: _IPNetworkAddr, exclude: _IPNetworkAddr
 ) -> tuple[list[IPNetwork], list[IPNetwork], list[IPNetwork]]: ...
 def spanning_cidr(ip_addrs: Iterable[_IPNetworkAddr]) -> IPNetwork: ...
-def iter_iprange(start: _IPAddressAddr, end: _IPAddressAddr, step: SupportsInt | SupportsIndex = 1) -> Iterator[IPAddress]: ...
+def iter_iprange(start: _IPAddressAddr, end: _IPAddressAddr, step: ConvertibleToInt = 1) -> Iterator[IPAddress]: ...
```

Comment:
I don't see any reason to make this change: I think `SupportsInt | SupportsIndex` expresses the idea of "int-like objects" better than `ConvertibleToInt` -- and I don't think it makes sense to pass something that isn't an "int-like object" to the `step` parameter here